package.json

```diff
@@ -3,7 +3,7 @@
   "version": "11.0.0",
   "description": "ESLint preset extending Airbnb, Flow, Prettier and Jest",
   "engines": {
-    "node": ">=10.13.0"
+    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

Comment:
We may consider also supporting non-LTS Node versions.

Reply:
Let's add `^13.0.0` and `^15.0.0`

Reply:
Updated based on https://nodejs.org/dist/latest-v16.x/docs/api/esm.html#modules-ecmascript-modules